README.md

```diff
@@ -2,9 +2,12 @@
 
 A docker based test framework
 
-This project packages the make targets that provide the tools and command shortcuts Plus3IT uses to develop and maintain our terraform modules.
+This project packages the make targets that provide the tools and command shortcuts
+Plus3IT uses to develop and maintain projects of all sorts.
 
-The makefile in this repository has been exposed as the entry point to the accompanying docker image. A list of the available make targets are provided below:
+The targets in this tardigrade-ci Makefile are included when using the accompanying
+docker image with your own project. A list of the available make targets are provided
+below:
```

Comment:
The list of targets has gotten out of date.  It's missing:

```
docker/build
docker/clean
docker/run
hcl/validate
```

Also, I should have corrected `eclint/lint` and made it `ec/lint`.  Would you consider adding `make init`?

Reply:
Updated using `make help`